src/components/TranslationAlertBox/TranslationAlertBox.tsx

```diff
@@ -0,0 +1,18 @@
+import { Alert, Box } from '@chakra-ui/react'
+import React from 'react'
+import { FaInfoCircle } from 'react-icons/fa'
+import { IconCircle } from 'components/IconCircle'
+import { Text } from 'components/Text'
+
+export const TranslationAlertBox: React.FC<{ translation: string }> = ({ translation }) => {
```

Comment:
I don't think that Translation is needed, I'm not the best at naming things but I would have used something like `InformationalAlert`